addons/web/static/src/views/record.js

```diff
@@ -8,7 +8,7 @@ const defaultActiveField = { attrs: {}, options: {}, domain: "[]", string: "" };
 
 class _Record extends Component {
     setup() {
-        const resModel = this.props.info.resModel;
+        let resModel = this.props.info.resModel;
```

Comment:
It's best to keep it as a const if it stays unchanged :)
```suggestion
        const resModel = this.props.info.resModel;
```